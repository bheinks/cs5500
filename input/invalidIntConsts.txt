0002147483648	
2147483648
-2147483648
9999999999
12345678901


