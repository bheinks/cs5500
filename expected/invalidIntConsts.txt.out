**** Invalid integer constant: 0002147483648
**** Invalid integer constant: 2147483648
TOKEN: T_MINUS     LEXEME: -
**** Invalid integer constant: 2147483648
**** Invalid integer constant: 9999999999
**** Invalid integer constant: 12345678901
